docs/how-to-guides/deploy.md

```diff
@@ -6,22 +6,78 @@ id: deploy
 
 ## Overview
 
-This guide shows you how to deploy any realm or package to the Gno chain. Deployment is be done by utilizing `gnokey`'s `maketx addpkg` API.
+This guide shows you how to deploy any realm or package to the Gno chain. It will
+show you how to:
+- Deploy Gno code in your browser via the Playground, 
+- Deploy Gno code from your local machine using `gnokey`'s `maketx addpkg` API.
+
+
+## Deployment via the Playground
+
+Deployment via the Playground is recommended for smaller realms and packages.
+For larger projects, it is recommended to write, test, and deploy your code from
```

Comment:
'...a local environment.' until the IDE is ready 🤞 (which we can callout, and in a way where we aren't suggesting it would replace local development, more as the first option or first entry)

Reply:
How about this? [4022313](https://github.com/gnolang/gno/pull/1766/commits/40223137bfe57e44e17d48039f61783f4f9a095e)